web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v2100/migrate-default.sql

```diff
@@ -10,10 +10,10 @@ DELETE FROM Settings WHERE id=80;
 
 -- New settings 
 INSERT INTO Settings VALUES (24,20,'securePort','8443');
-INSERT INTO Settings VALUES (956,1,'hidewithheldelements',NULL);
-INSERT INTO Settings VALUES (957,956,'enable','false');
-INSERT INTO Settings VALUES (958,956,'keepMarkedElement','true');
-INSERT INTO Settings VALUES (955,956,'ignored','true');
+INSERT INTO Settings VALUES (1000956,1,'hidewithheldelements',NULL);
+INSERT INTO Settings VALUES (1000957,1000956,'enable','false');
+INSERT INTO Settings VALUES (1000958,1000956,'keepMarkedElement','true');
+--INSERT INTO Settings VALUES (1000955,1000956,'ignored','true');
```

Comment:
Same with this: something specific for your usecase, or is there a reason for this?